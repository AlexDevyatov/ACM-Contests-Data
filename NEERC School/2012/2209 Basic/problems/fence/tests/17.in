7
-26 -23
-8 -29
21 -25
28 -20
27 12
-2 23
-17 19
0 13
