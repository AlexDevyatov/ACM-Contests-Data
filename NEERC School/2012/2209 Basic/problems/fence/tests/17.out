8
-26 -23
-8 -29
21 -25
28 -20
27 12
-2 23
0 13
-17 19
